editor/src/clj/editor/welcome.clj

```diff
@@ -486,7 +495,20 @@
       (b/bind! (location-field-title-property new-project-location-field) sanitized-title-property))
     (doto template-list
       (ui/cell-factory! (fn [project-template]
-                          {:graphic (make-template-entry project-template localization)})))
+                          {:graphic (make-template-entry project-template localization)}))
+      (.setOnMouseClicked (ui/event-handler event
+                            (let [^MouseEvent mouse-event event
+                                  ^Node target (.getTarget mouse-event)]
+                              (when (and (instance? javafx.scene.control.ListCell target)
+                                         (nil? (.getItem ^ListCell target)))
+                                (.clearSelection (.getSelectionModel template-list)))
+                              (when (and (= 2 (.getClickCount mouse-event))
+                                         (not-empty (ui/selection template-list)))
+                                (.fire create-new-project-button)))))
+      (.setOnKeyPressed (ui/event-handler event
+                          (when (= javafx.scene.input.KeyCode/ENTER (.getCode ^KeyEvent event))
+                            (when-some [project-template (first (ui/selection template-list))]
```

Comment:
`project-template` is unused, so you could combine the check with the outer `when`?